### ISSUE TITLE
When creating a client role with slash mark(s) in the name, an invalid value is indicated in the `Location`

### ISSUE THREAD

**Issue by galoitsu** [keycloak/keycloak#28203]
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

admin/api

### Describe the bug

I am using Admin API.
When creating a client role with slash mark(s) in the name, an invalid value is indicated in the `Location`.
(I cannot use value indicated in the `Location`.)

### Version

24.0.2

### Regression

- [ ] The issue is a regression

### Expected behavior

URL-encoded
```
Location: http://localhost:8080/admin/realms/master/clients/fcd9c9d9-a21f-4480-afd2-7d96aa0d6c0f/roles/name%2Fwith%2Fslash
```

OR

Using client role id(UUID)
```
Location: http://localhost:8080/admin/realms/master/roles-by-id/80c36fbd-5c42-4338-97e9-3170dc31ee0a
```

### Actual behavior

```
Location: http://localhost:8080/admin/realms/master/clients/fcd9c9d9-a21f-4480-afd2-7d96aa0d6c0f/roles/name/with/slash
```

### How to Reproduce?

1. Create a client role with slash mark(s) in the name
  ```
  $ curl \
    -i \
    -X 'POST' 'http://localhost:8080/admin/realms/master/clients/fcd9c9d9-a21f-4480-afd2-7d96aa0d6c0f/roles' \
    -H "Authorization: Bearer $access_token" \
    -H 'Content-Type: application/json' \
    -d '{"name": "name/with/slash"}'
  HTTP/1.1 201 Created
  Location: http://localhost:8080/admin/realms/master/clients/fcd9c9d9-a21f-4480-afd2-7d96aa0d6c0f/roles/name/with/slash
  Referrer-Policy: no-referrer
  Strict-Transport-Security: max-age=31536000; includeSubDomains
  X-Content-Type-Options: nosniff
  X-Frame-Options: SAMEORIGIN
  X-XSS-Protection: 1; mode=block
  content-length: 0
  ```

2. Get `Location` and use it
  ```
  $ curl \
    -i \
    -X 'GET' 'http://localhost:8080/admin/realms/master/clients/fcd9c9d9-a21f-4480-afd2-7d96aa0d6c0f/roles/name/with/slash' \
    -H "Authorization: Bearer $access_token"
  HTTP/1.1 404 Not Found
  content-length: 118
  Content-Type: application/json
  Referrer-Policy: no-referrer
  Strict-Transport-Security: max-age=31536000; includeSubDomains
  X-Content-Type-Options: nosniff
  X-Frame-Options: SAMEORIGIN
  X-XSS-Protection: 1; mode=block

  {"error":"HTTP 404 Not Found","error_description":"For more on this error consult the server log at the debug level."}
  ```

**Comment by graziang** [comment]
Hi @galoitsu, this should be already fixed by #27542

**Comment by galoitsu** [comment]
@graziang
Sorry... I overlooked that it had already been fixed, even though it hadn't been released yet.
Thank you for your response.
